docs/content/preview/tutorials/azure/azure-app-service.md

```diff
@@ -0,0 +1,138 @@
+
+First, visit GitHub for the [web application](https://github.com/YugabyteDB-Samples/yugabytedb-azure-app-service-demo-nodejs) you will be deploying to Azure App Service.
+
+Here's the application in action, after deploying to Azure App Service.
+
+![Azure App Service web application displaying shoe inventory and recent sales from YugabyteDB](/images/tutorials/azure/azure-app-service/yb-shoes.png "Azure App Service web application displaying shoe inventory and recent sales from YugabyteDB")
+
+Let's cover the components required to build and deploy this application.
+
+## Create a web app on Azure App Service
+
+With Azure App Service, developers can deploy applications in a variety of programming languages and application architectures. For YB Shoes, we will create a web app using Node.js.
+
+From the Azure portal, create a web app with Node.js as the runtime stack. This allows Azure to configure the runtime environment with Node.js and the dependencies required to run our application.
+
+![Create a web app with the Node.js runtime in Azure App Service.](/images/tutorials/azure/azure-app-service/azure-web-app.png "Create a web app with the Node.js runtime in Azure App Service.")
+
+As we're deploying our application to Azure from a local repository on our machine, we can forego all settings related to continuous deployment and GitHub Actions. However, for production applications, these integrations greatly improve the development process by deploying applications to Azure automatically when the repository is updated, without the need for manual intervention.
+
+Additionally, by selecting a free or basic tier of Azure App Service, we will deploy our application to a single zone by default. Premium pricing plans allow for multi-zone deployments for increased availability.
+
+After creating your application, visit the associated URL to confirm it's up and running!
+
+![Application running in Azure and waiting for content.](/images/tutorials/azure/azure-app-service/azure-app-running.png "Application running in Azure and waiting for content.")
+
+Now that we've set the foundation on Azure, let's deploy a database cluster for YB Shoes on YugabyteDB Managed.
+
+## Get Started on YugabyteDB Managed
+
+Visit our documentation site to create a YugabyteDB cluster on Azure.
+
+I've chosen to deploy a three-node cluster to Azure, in the WestUS3 region. This configuration provides fault tolerance across availability zones. However, you can start with an always-free single-node cluster.
+
+![Create free YugabyteDB single-region cluster.](/images/tutorials/azure/azure-app-service/yb-create-cluster.png "Create free YugabyteDB single-region cluster.")
+
+Add the [outbound addresses for your web app](https://learn.microsoft.com/en-us/azure/app-service/overview-inbound-outbound-ips), which can be found in the networking tab of the Azure portal. This will ensure that a connection can be made between Azure App Service and YugabyteDB.
```

Comment:
Is this the IP allow list part of the create cluster wizard your are talking about? Should probably do discreet steps in this section as well